daemon/algod/server.go

```diff
@@ -297,9 +297,10 @@ func (s *Server) Start() {
 
 	addr = listener.Addr().String()
 	server = http.Server{
-		Addr:         addr,
-		ReadTimeout:  time.Duration(cfg.RestReadTimeoutSeconds) * time.Second,
-		WriteTimeout: time.Duration(cfg.RestWriteTimeoutSeconds) * time.Second,
+		Addr:           addr,
+		ReadTimeout:    time.Duration(cfg.RestReadTimeoutSeconds) * time.Second,
+		WriteTimeout:   time.Duration(cfg.RestWriteTimeoutSeconds) * time.Second,
+		MaxHeaderBytes: 4096,
```

Comment:
According to your wikipedia link, this is not a magic number:
> A unique value with unexplained meaning or multiple occurrences which could (preferably) be replaced with a named constant

The number is assigned to a clearly defined value and there is exactly 1 occurance.

> A constant numerical or text value used to identify a [file format](https://en.wikipedia.org/wiki/File_format) or protocol; for files, see [List of file signatures](https://en.wikipedia.org/wiki/List_of_file_signatures)

This is not some cross system value, just a number of bytes.

>  A distinctive unique value that is unlikely to be mistaken for other meanings (e.g., [Globally Unique Identifiers](https://en.wikipedia.org/wiki/Globally_Unique_Identifier))

There is no mistaking what this number is trying to accomplish.


It seems to me that this suggestion is about the letter of the rule but completely misses the spirit. The constant is used in exactly one place to assign a value to a variable with nice documentation.

I'll add a comment above the assignment that describes why the value was chosen. I did wonder why it was chosen for wsNetwork.

Reply:
The wsNetwork comment is copy/pasted from the http.MaxHeaderBytes comment. It would be nice if the comment were changed to describe why the value is 4096.

Reply:
I don't care enough to debate this point, so I've made the change. As I mentioned, while this suggestion may indeed follow the letter of the rule, it misses the spirit.